benchbuild/utils/container.py

```diff
@@ -61,7 +61,7 @@ def is_valid_container(path):
         tmp_hash_path = __CONTAINER_DEFAULT__ + ".hash" 
         tmp_hash_file = open(tmp_hash_path, 'r')
         tmp_hash = tmp_hash_file.readline() 
-    except:
+    except IOError:
```

Comment:
Rethink your message, we _might_ not be able to open the file, which is not the same as 'does not exist'.